velox/common/base/SimdUtil.h

```diff
@@ -354,7 +354,12 @@ xsimd::batch<T, A> iota(const A& = {});
 template <typename T, typename A = xsimd::default_arch>
 xsimd::batch<T, A> setAll(T value, const A& = {}) {
   if constexpr (std::is_same_v<T, bool>) {
+#if defined(__aarch64__)
+    return xsimd::batch<T, A>(
+        xsimd::broadcast<bool, A>((bool)(value ? -1 : 0)));
```

Comment:
As we discussed in #3487 let us put a test to make sure the bits are all set:

```c++
auto bits = setAll(true);
auto words = reinterpret_cast<int64_t*>(&bits);
for (int i = 0; i < xsimd_batch<int64_t>::size; ++i) {
  EXPECT_EQ(words[i], -1ll);
}
```

Reply:
Thanks @Yuhta. I will add this unit test and confirm if my implementation is correct or not.

Reply:
Fixed the implementation and uploaded the code. Thank you.